code/modules/uplink/uplink_items.dm

```diff
@@ -2328,3 +2337,440 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
+				/mob/living/simple_animal/hostile/russian/army/army3,
+				/mob/living/simple_animal/hostile/russian/army/army4)
+	Stationwide_charges = 1
+	var/restricted = TRUE
+
+/obj/item/backup_caller/soviet/Initialize()
+	. = ..()
+	playsound(get_turf(src), 'sound/items/russianbackupcaller.ogg', 60, 0, 0)
+
+/obj/item/backup_caller/soviet/attack_self(mob/living/user)
+	var/list/factions = usr.faction
+	if(restricted && !(ROLE_SYNDICATE in factions))
+		return to_chat(user, "<span class='warning'>No response!</span>")
+	if(Stationwide_mode && cooldown < world.time)
+		to_chat(user, "<span class='bold'>\"The power is in numbers.\"</span>")
+		playsound(get_turf(src), 'sound/items/russianbackupcaller.ogg', 60, 0, 0)
+	. = ..()
+
+/obj/item/backup_caller/soviet/drop()
+	var/chosen_mob = pick(mobs)
+	var/mob/living/simple_animal/hostile/russian/army/M = new chosen_mob
+	M.defend_target = usr
+	var/list/factions = usr.faction
+	if(ROLE_SYNDICATE in factions)
+		M.faction = list(ROLE_SYNDICATE)
+	else
+		M.faction = factions
+	var/turf/T = get_turf(usr)
+	var/turf/R = locate(T.x + rand(-3,3), T.y + rand(-3,3), T.z)
+	if(iswallturf(R) || !R)
+		R = get_turf(usr)
+	var/obj/structure/closet/supplypod/bluespacepod/pod = new()
+	pod.explosionSize = list(0,0,0,0)
+	M.forceMove(pod)
+	new /obj/effect/pod_landingzone(R, pod)
+
+/obj/item/backup_caller/soviet/unrestricted
+	restricted = FALSE
+
+/obj/item/stabbing_license
+	name = "Stabbing License"
+	desc = "A card provided by Space London. An unknown technology will make people around you know that you are, infact, allowed to use sharp items for legal stabbing."
+	icon = 'icons/obj/card.dmi'
+	icon_state = "slicense2"
+	lefthand_file = 'icons/mob/inhands/equipment/idcards_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/idcards_righthand.dmi'
+	item_state = "orange_id"
+	var/cooldown = 0
+	var/cooldown_time = 60
+
+/obj/item/stabbing_license/examine(mob/user)
+	. = ..()
+	. += "<span class='notice' Holding it in hand will make all your attacks with sharp items more powerful.</span>"
+
+/obj/item/stabbing_license/attack_self(mob/user)
+	if(cooldown > world.time)
+		return
+	cooldown = world.time + cooldown_time
+	user.visible_message("<span class='userdanger'>[user] shows you their [src], mate! You got no balls to mess with them in stabbing fight, innit?</span>", "<span class='suicide'>You show your [src] to lower the morale of everyone around, innit?</span>")
+
+/obj/item/stabbing_license/worn_overlays(isinhands)
+	. = ..()
+	if(isinhands)
+		. += mutable_appearance('icons/effects/effects.dmi', "blessed", MOB_LAYER + 0.01)
+		var/obj/item/I
+		if(I.sharpness == IS_SHARP || I.sharpness == IS_SHARP_ACCURATE)
+			. -= mutable_appearance('icons/effects/effects.dmi', "blessed", MOB_LAYER + 0.01)
+			. += mutable_appearance('icons/effects/cult_effects.dmi', "bloodsparkles", MOB_LAYER + 0.01)
+			I.attack_weight = I.attack_weight + 0.5
+			I.force = I.force + 3
```

Comment:
Why is this in the uplink?

Reply:
I didn't know where else to put it

Reply:
I guess I should make a new file for them then

Reply:
nevermind I forgot to add it to devices